tool/jt.rb

```diff
@@ -43,6 +43,22 @@
 METRICS_REPS = Integer(ENV['TRUFFLERUBY_METRICS_REPS'] || 10)
 DEFAULT_PROFILE_OPTIONS = %w[--cpusampler --cpusampler.Output=flamegraph]
 
+MRI_TEST_RETAG_RETAIN_PATTERNS = [
+  /hangs/i,
+  /slow/i,
+  /java\.lang\./,
+  /retain-on-retag/,
+  /OOM/,
+  /RubyVM/,
+  /ShouldNotReachHere/,
+  /LLVMLinkerException/,
```

Comment:
Could you remove this? It's a Sulong exception so shouldn't happen anymore and there are 0 matches of it